rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -22,158 +22,184 @@
+        caffe_reader_meta_data = b.caffeReaderDetection(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     else:
         Pipeline._current_pipeline._reader = "CaffeReader"
-        caffe_reader_meta_data = b.caffeReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+        caffe_reader_meta_data = b.caffeReader(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
 
     if (bbox == True):
-        return (caffe_reader_meta_data,bboxes, labels)
+        return (caffe_reader_meta_data, bboxes, labels)
     else:
         return (caffe_reader_meta_data, labels)
 
-def caffe2(*inputs, path, bbox=False, additional_inputs=0, bytes_per_sample_hint=0, image_available=True,
-           initial_fill=1024, label_type=0, lazy_init=False, num_labels=1,  num_shards=1, pad_last_batch=False,
-           prefetch_queue_depth=1, preserve=False, random_shuffle=False, read_ahead=False, seed=-1, shard_id=0,
-           skip_cached_images=False, stick_to_shard=False, tensor_init_bytes=1048576, device=None):
 
-    #Output
+def caffe2(path, bbox=False):
+
+    # Output
     bboxes = []
     labels = []
-    kwargs_pybind = {"source_path": path, "is_output":True}
+    kwargs_pybind = {"source_path": path, "is_output": True}
     if (bbox == True):
         Pipeline._current_pipeline._reader = "Caffe2ReaderDetection"
-        caffe2_meta_data = b.caffe2ReaderDetection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+        caffe2_meta_data = b.caffe2ReaderDetection(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     else:
         Pipeline._current_pipeline._reader = "Caffe2Reader"
-        caffe2_meta_data = b.caffe2Reader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+        caffe2_meta_data = b.caffe2Reader(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     if (bbox == True):
-        return (caffe2_meta_data,bboxes, labels)
+        return (caffe2_meta_data, bboxes, labels)
     else:
         return (caffe2_meta_data, labels)
 
-def video(*inputs, sequence_length, additional_decode_surfaces=2, bytes_per_sample_hint=0, channels=3,
-          dont_use_mmap=False, dtype=types.FLOAT, enable_frame_num=False,  enable_timestamps=False, file_list="",
-          file_list_frame_num=False, file_list_include_preceding_frame=False, file_root="", filenames=[],
-          image_type=types.RGB, initial_fill=1024, labels="", lazy_init=False, normalized=False, num_shards=1,
-          pad_last_batch=False, pad_sequences=False, prefetch_queue_depth=1, preserve=False, random_shuffle=False,
-          read_ahead=False, seed=-1, shard_id=0, skip_cached_images=False, skip_vfr_check=False, step=1,
-          stick_to_shard=False, stride=1, tensor_init_bytes=1048576, decoder_mode=types.SOFTWARE_DECODE,
-          device=None, name=None):
+
+def video(sequence_length, file_list_frame_num=False, file_root="", image_type=types.RGB, num_shards=1,
```

Comment:
Required
enable_framenum
enable_timestamps

Will be needed in future
file_list
file_list_include_preceding_frame
normalized
pad_sequences
skip_vfr_check

Reply:
Please address this comment